### PR TITLE
Chapter9 exercise3

### PR DESCRIPTION
**Pull request by Jenerth22Rodriguez**
- [x] we have seen how to communicate information from the calling function to the called function. 
- [x] To send information in the other direction, you use the function return value.
- [x] we’ll construct a function that returns the smaller of its two arguments.
- [x] We’ll call the function lesser() because it’s designed to handle int values.

### CODE REVIEW

File: lesser.c [hunk 1]
```diff
@@ -0,0 +1,27 @@
+#include <stdio.h>
+
+static int lesser(int name, int name2);
+
+int main(void)
+{
+int evil1, evil2;
```
Comment (ryanplusplus) [inline lesser.c]:
```suggestion
   int evil1, evil2;
```

**Comment by ryanplusplus** [comment]
Is this from the book? Can you add information about the exercise?

**Comment by ryanplusplus** [comment]
What did you learn when doing this exercise?

File: lesser.c [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+   while(scanf("%d %d", &evil1, &evil2) == 2)
+   {
+      printf("The lesser of %d and %d is %d.\n", evil1, evil2, lesser(evil1, evil2));
+      printf("Enter a pair of integers (q to quit):\n");
+   }
+   printf("Bye.\n");
+   return 0;
+}
+
+int lesser(int firstNumber, int secondNumber)
```
Comment (StevenRoot) [inline lesser.c]: static? -- also if move this routine above main in the file and call it static you don't need to waste time with the additional static prototype

File: lesser.c [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+{
+   int evilNumber;
+   if(firstNumber < secondNumber)
+      evilNumber = firstNumber;
+   else
+      evilNumber = secondNumber;
```
Comment (StevenRoot) [inline lesser.c]: Can you add appropriate braces please  {. } etc.

**Pull request merged**